docker/incremental_load.sh.tpl

```diff
@@ -18,7 +18,7 @@ set -eu
 
 # This is a generated file that loads all docker layers built by "docker_build".
 
-RUNFILES="${PYTHON_RUNFILES:-${BASH_SOURCE[0]}.runfiles}"
+RUNFILES="${PYTHON_RUNFILES:-${PWD}/${BASH_SOURCE[0]}.runfiles}"
```

Comment:
Humm this might not work. Let say I invoke it by absolute path?

Maybe instead use some kind of magic to get the good path like realpath? maybe simply:

`$(cd ${BASH_SOURCE[0]}.runfiles && pwd -P)`?

Reply:
I put it in a function, should be RFAL.